### ISSUE TITLE
Pre-trained Discriminator model

### ISSUE THREAD

**Issue by compressor1212** [jik876/hifi-gan#136]
Can I have a pre-trained discriminator of the model so that I can use it for the feature loss calculation, that will save my time?

**Comment by katya-bateeva** [comment]
You can use discriminator from a pretrained universal model, you may download it [here](https://drive.google.com/drive/folders/1-eEYTB5Av9jNql0WGBlRoi-WH2J7bp5Y). The discriminator is the same for all versions (V1, V2, V3) of generator.
